Level_1/failure.py

```diff
@@ -0,0 +1,17 @@
+def solution(N, stages):
+    arr = []
+    clear = 0
+    for i in range(N, -1, -1):
+        failure = stages.count(i + 1)
+        clear += failure
+        if clear:
+            arr.append(failure / clear)
+        else:
+            arr.append(0)
```

Comment:
단순 if else 는 이렇게 사용하는게 더 좋을 듯 합니다~!:grinning:
```suggestion
        arr.append(failure / clear if clear else 0)
```

Reply:
파이썬 한줄 쓰기를 더 연습해야 겠네요.